content/en/_index.md

```diff
@@ -36,99 +36,99 @@ section2:
       children:
         - content: Improve your environment utilization and reduce internal infrastructure costs
         - content: Support multi-cluster and multi-cloud Kubernetes management, avoiding vendor lock-in
-        - content: Provides security enhancements, supports multiple storage and network solutions
-        - content: Fully trusted, delivers a certified Kubernetes platform and distribution
+        - content: Provide security enhancements, multiple storage and network solutions
+        - content: Deliver a trustworthy and certified Kubernetes platform and distribution
 
     - name: Developers
-      content: Focus on your bussiness, others are run in automated tools
+      content: Focus on your business as automation tools do the rest
       icon: /images/home/74.png
       children:
-        - content: Smooth user experience, reduce the learning curve of the cloud native stack
+        - content: Create a smooth user experience and reduce the learning curve of the cloud native stack
         - content: Provide toolkits and deployment automation tailored to any application environment
-        - content: Out-of-box logging, monitoring and multi-tenancy, improving development efficiency
+        - content: Offer out-of-box logging, monitoring and multi-tenancy functions, improving development efficiency
         - content: Support application lifecycle management, accelerating time to market
 
     - name: Ops Team
-      content: Building one-stop enterprise-grade DevOps framework
+      content: Build a one-stop enterprise-grade DevOps framework
       icon: /images/home/71.svg
       children:
-        - content: Centralized log collection, monitoring and alerting from infrastructure to applications.
-        - content: Streamlined continuous deploy, test, release, upgrade and scale
-        - content: Better track, route and optimize communications within Kubernetes for cloud native apps
-        - content: Easy-to-use web terminal and graphical panel, satisfying the habits of different users 
+        - content: Centralized log collection, monitoring and alerting from infrastructure to applications
+        - content: Streamlined process of continuous deployment, test, release, upgrade and scaling
+        - content: Better tracking, routing and optimized communications within Kubernetes for cloud native apps
+        - content: Easy-to-use web terminal and graphical panel, satisfying the needs of different users 
 
 section4:
   title: Key Features
-  content: If you want to use an open source project, but act like a commercial product, KubeSphere is your choice. <br>The Roadmap listed the planning features, you can raise a proposal to submit your ideas with us.
+  content: If you are seeking for an open source project that also provides commercial products, KubeSphere is your choice. <br>The Roadmap lists planned features and you can raise a proposal to contribute your ideas to the community.
```

Comment:
`If you are seeking for an open source project that also provides commercial products` is not exactly what I mean

Reply:
Changed to: If you are seeking for a open source project run as a commerical product